homeassistant/components/zwave_me/lock.py

```diff
@@ -0,0 +1,51 @@
+"""Representation of a doorlock."""
+from typing import Any
```

Comment:
Need to import annotations also
```suggestion
"""Representation of a doorlock."""
from __future__ import annotations

from typing import Any
```